frontend/packages/dev-console/src/components/project-access/ProjectAccessForm.tsx

```diff
@@ -16,39 +20,58 @@ const ProjectAccessForm: React.FC<ProjectAccessFormProps> = ({
   errors,
   dirty,
   roles,
+  roleBindings,
+  values,
 }) => {
   const { t } = useTranslation();
+  const [isStaleInfo, setIsStaleInfo] = React.useState<boolean>(false);
   const disableSubmit = !dirty || !_.isEmpty(errors);
+
+  React.useEffect(() => {
+    if (!_.isEqual(roleBindings, values)) {
+      setIsStaleInfo(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [roleBindings]);
+
+  const onReload = () => {
+    handleReset();
+    setIsStaleInfo(false);
+  };
+
```

Comment:
```suggestion
 const disableSubmit = !dirty || !_.isEmpty(errors);
 const isStaleInfo = !_.isEqual(roleBindings, values);
```

Reply:
@divyanshiGupta  The issue that I am trying to address here is, if the user has 2 tabs open and updates the role binding in tab 1 then the stale info alert should be shown in tab 2. The user should reload and fetch the latest list. The stale info alert is shown only when the project access list is updated other wise if the user edits the form it should still show `you-have-made-changes..`

I have updated this PR. Now, if the list is stale, the Save button is disabled. The user must reload the page to be able to add new role bindings

Reply:
I am aware about that but I didn't understand what it has to do with the changes that I suggested. The above changes will do the same thing with a single line addition. Have you tried it out? Let me know if it doesn't work for you.

Reply:
@divyanshiGupta Yes, I have tried it out. with the above changes, when I try to do `add access` it shows the stale info alert instead of `you-have-made-changes` alert

Reply:
Okay in that you instead of using useEffect and also mainting a state you can simply use useMemo.